### DISCUSSION TITLE
Relation Column

### DISCUSSION THREAD

**Discussion by tomasnorre** [filamentphp/filament#421] [Help]
I have a problem with my Laravel filaments Relation in the "backend" interface. The column with my Relation isn't shown.

I have a simple model with a GolfCourse and Country. The edit is working and updating the Database as expected, but I cannot get it to list it on index page of the Filament backend.

I have following in my `GolfCourseResources.php`

```php
 public static function table(Table $table)
    {
        return $table
            ->columns([
                Columns\Text::make('name'),
                Columns\Column::make('country.name'),
            ])
            ->filters([]);
    }
``` 
According to documentation, this should be how it done. https://filamentadmin.com/docs/tables/#columns

```shell
And get the error:

InvalidArgumentException
View [cells.column] not found. (View: .../vendor/filament/filament/packages/tables/resources/views/components/table.blade.php)
http://127.0.0.1:8000/admin/resources/golf-courses

cells.column was not found.

Are you sure the view exists and is a .blade.php file?
```
Do you have any hints on this?

**Comment by tomasnorre** [comment]
@telkins I saw you have a question "similar" to this, would you happen to know the answer to my question?

**Reply by ryangjchandler** [reply]
`Column` isn't a valid column type. The documentation uses that purely as an API example for all columns. 

If you use a text column, it should work.

**Reply by telkins** [reply]
> @telkins I saw you have a question "similar" to this, would you happen to know the answer to my question?

Hi, @tomasnorre,

Not sure I see the similarities, but it sounds like @ryangjchandler gave you a good starting point...that `Column` isn't actually a valid column.  Hopefully that's the only problem. 🤓

Let us know if changing it to a correct column type resolves your problem. 👍 

Travis

**Reply by tomasnorre** [reply]
Thanks @ryangjchandler, I misunderstood that. 
Works like a charm.
